2014-06-23  Marek Polacek  <polacek@redhat.com>

	* g++.old-deja/g++.brendan/crash64.C: Remove dg-warning.
	* g++.old-deja/g++.brendan/overload8.C: Likewise.

--- gcc/testsuite/g++.old-deja/g++.brendan/crash64.C	(revision 211824)
+++ gcc/testsuite/g++.old-deja/g++.brendan/crash64.C	(working copy)
@@ -16,4 +16,4 @@
 
 static _type_desc _type_metatype("metatype", sizeof(metatype),
   (RF_Ptr)0, 0, 1, 1,
-  _im_pers_mem_spec( ((size_t)&((( metatype *)0)-> base_list )) , 1)); // { dg-warning "" } cannot pass non-pod
+  _im_pers_mem_spec( ((size_t)&((( metatype *)0)-> base_list )) , 1));
--- gcc/testsuite/g++.old-deja/g++.brendan/overload8.C	(revision 211824)
+++ gcc/testsuite/g++.old-deja/g++.brendan/overload8.C	(working copy)
@@ -6,4 +6,4 @@
 void zxcvbnm(int n,...){n=1;}
 int main(){complex c; Complex C;
 zxcvbnm(1,c);
-zxcvbnm(1,C);} // { dg-warning "" } cannot pass non pod
+zxcvbnm(1,C);}
